src/Config/Converters/DataSourceFilesConverter.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Azure.DataApiBuilder.Config.ObjectModel;
+
+namespace Azure.DataApiBuilder.Config.Converters;
+
+/// <summary>
+/// Converter for DataSourceFiles
+/// </summary>
+class DataSourceFilesConverter : JsonConverter<DataSourceFiles>
+{
+    /// <inheritdoc/>
+    public override DataSourceFiles? Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
```

Comment:
what is `typeToConvert` being used for in this method? just added to overload the right signature?

Reply:
Yes.